x/icacallbacks/module_ibc.go

```diff
@@ -0,0 +1,149 @@
+package icacallbacks
```

Comment:
Why add this file?

Reply:
hmmmm probably this is from the composable PR, i think i migrated it by mistakte to my new 47 branch, will remove